### PR TITLE
LPS-X hibernate 12 14

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the pr is based on https://github.com/dantewang/liferay-portal/pull/2090 and then rebased on master.
I add the last commit as workaround.

Should I send it to liferay-core-infra?

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yuhai** [comment]
ci:test

File: modules/apps/change-tracking/change-tracking-store-api/bnd.bnd [hunk 1]
```diff
@@ -5,4 +5,10 @@ Export-Package:\
 	com.liferay.change.tracking.store.exception,\
 	com.liferay.change.tracking.store.model,\
 	com.liferay.change.tracking.store.service,\
-	com.liferay.change.tracking.store.service.persistence
\ No newline at end of file
+	com.liferay.change.tracking.store.service.persistence
+Import-Package:\
+	javassist.util.proxy;version="3.28.0",\
```
Comment (dantewang) [inline modules/apps/change-tracking/change-tracking-store-api/bnd.bnd:10]: @yuhai Actually you don't need to specify the exact version. Bnd will automatically calculate a range based on what is on compile class path during build.

**Comment by dantewang** [comment]
@yuhai The integration test failures should be real. For service builder tests you may also need to add the import-package to service-builder-test-api. For other failures, maybe something is wrong with the transaction executor handling...

**Comment by yuhai** [comment]
resend it to https://github.com/dantewang/liferay-portal/pull/2126

**Pull request closed without merging**